### DISCUSSION TITLE
[4.0] Change Order of Menu Item Types for Articles

### DISCUSSION THREAD

**Discussion by 200MPHMEDIA** [joomla/joomla-cms#47175] [2. Cold Features]
### Steps to reproduce the issue
When creating a Menu Link Item for an Article, when the "Articles" selector is opened, "Archived Articles" (likely the least used), appears at the top, while "Articles" (likely the MOST used), is at the bottom and it is necessary to scroll down to it.

Is it remotely possible that "Articles" (the most used option), be moved to being the first item on the list of Menu Link Item choices? It just makes sense.

It has always been a nuisance in previous versions, and there is no reason to perpetuate it forward to 4.0 is it?

Can someone on the 4.0 team kick that into action please.

**Comment by brianteeman** [comment]
The list is displayed in alphabetical order

**Comment by 200MPHMEDIA** [comment]
Should the list not be ordered for “convenience” vs fixed.

 

The need to scroll down each time for Articles is not necessary.

 

Change the name from “Single Article” to “Article-Single” would put it near the top.

 

Changing “Archived Articles” to “Articles-Archived” would order those two items correctly.

**Comment by infograf768** [comment]
These naming depend on the language used. What may be OK in English is not necessary correct in other languages.
This should be closed.

**Comment by brianteeman** [comment]
The only thing I could think of would be to change the name so that it has a numerical prefix. That _should_ work in all languages. eg 2.. Archived Articles 1. Categpry Blog

Or you could do it yourself with language overrides as I just did here

![image](https://user-images.githubusercontent.com/1296369/72205543-410a3b00-347c-11ea-9120-057cf9cb8cdc.png)
,

**Comment by 200MPHMEDIA** [comment]
Does the back end not change based on language.

 

In any case, thanks for the push back on helping with the issue.

 

Guess we just need to “live with it” and keep scrolling down for the most used choice.

**Comment by 200MPHMEDIA** [comment]
Numbering might make sense.

 

Problem with “overrides” reverts back to the question of: how many beginner users know how to create/invoke overrides?  So that does not appear to be a proper solution.

 

In any case, any way that the MOST USED (Articles) should appear on a list before the LEAST USED (Archived Articles) makes sense.

**Comment by infograf768** [comment]
File concerned is com_content.sys.ini
We do not have the issue in French

en-GB
COM_CONTENT_ARCHIVE_VIEW_DEFAULT_TITLE="Archived Articles"
COM_CONTENT_ARTICLE_VIEW_DEFAULT_TITLE="Single Article"

fr-FR
COM_CONTENT_ARTICLE_VIEW_DEFAULT_TITLE="Article"
COM_CONTENT_ARCHIVE_VIEW_DEFAULT_TITLE="Articles archivés"
as we thought we did not need to use the word "single" ("seul") in French which would not change the order anyway but looks quite useless.

Therefore you can make an override for en-GB or we can modify in core en-GB by taking off the word "Single"

Adding a numerical prefix only for articles would make no sense imho as we should be consequent and also add such prefixes for other components. What would be the correct order for Users?

**Comment by brianteeman** [comment]
Removing the word Single will not change the alphabetical order ;)

Archived Articles
Article

**Comment by brianteeman** [comment]
Note that in Farsi - com_content is the penultimate set of options when selecting a menu

![image](https://user-images.githubusercontent.com/1296369/72245925-8730dd00-35e9-11ea-8852-4398f8e90154.png)

**Comment by infograf768** [comment]
> Removing the word Single will not change the alphabetical order ;)

It will in the sense that it would display near the top instead of the end of the list.

**Comment by brianteeman** [comment]
It is not a good solution. 
There is an ordering column in the table - its just not used :(

**Comment by brianteeman** [comment]
The ordering could then be made consistent across all languages

**Comment by infograf768** [comment]
Not sure about that as ordering depends on db collation.

**Comment by brianteeman** [comment]
Really. We use the ordering column everywhere

**Comment by infograf768** [comment]
ordering up an down does not mean it is the ordering related to language. It is only relative to alphabetical order as the collation sees it.

**Comment by brianteeman** [comment]
I am talking about using the ordering column which is numerical.
